tests/test_predicates.py

```diff
@@ -144,6 +105,6 @@ def test_lift_matrix(self):
         self.assertTrue(equal(target, lift_matrix(base, list_test)))
         self.assertTrue(equal(target, lift_matrix(base, vector_test)))
         self.assertTrue(equal(target, lift_matrix(base, matrix_test)))
-    '''
```

Comment:
Agree with Joe, there are a lot of undefined names here, including some of the things like `ccw`, `incircle`, etc.